### DISCUSSION TITLE
system reboot from sudosignals

### DISCUSSION THREAD

**Discussion by mikkokunnari** [SudoMagicCode/SudoSignals#15] [v1 - RFE & Feature Requests]
system reboot from sudosignals. Now it can be done from touchdesigner with python , but would be good to have it on sudosignals if touchdesigner refuses to start. Also to work it on schedule too.

**Comment by IanShelanskey** [comment]
Currently, you can stop and start processes from the sudoSignals dashboard so you can restart TD if something goes wrong. Are you asking for a full machine reboot?

**Reply by mikkokunnari** [reply]
Im asking for full machine reboot. :)
